scripts/activate-snyk-project.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+SNYK_TOKEN=$1
+IMAGE_ID=$2
+TAG=$3
+COMMIT_HASH=$4
+TAG_WITH_COMMIT_HASH="dwolla/$IMAGE_ID:$TAG-$COMMIT_HASH"
+
+function get_snyk_projects() {
+    curl \
+    --silent \
+    -H "Authorization: token $SNYK_TOKEN" \
+    https://snyk.io/api/v1/org/dwolla/projects
+}
+
+function activate_snyk_project() {
+    PROJECT_ID=$1
+    curl \
+    -X POST \
+    --silent \
+    -H "Authorization: token $SNYK_TOKEN" \
```

Comment:
Unfortunately, I didn't have much luck using a .netrc here instead of sending an authorization header. I think this is because the SNYK_TOKEN is technically an API token vs. an Oauth Token.

Reply:
If we used an environment variable here, would it print the value out when it executes? That might be a good compromise. (I think you're right that `.netrc` can't be used for bearer tokens—AFAICT it only works for basic auth (user/pass)

Reply:
good idea! [This commit](https://github.com/Dwolla/docker-java/pull/10/commits/bfea745e856fbf861da1f4d6a22a8214571c6994) should pass in SNYK_TOKEN as an env var to the `activate-snyk-project` script.

Out of curiosity, where do you see the auth header value being printed out when curl executes? I thought `--silent` would prevent any output from printing.